tests/test_brownian_interval.py

```diff
@@ -0,0 +1,311 @@
+        pytest.skip(msg="CUDA not available.")
+
+    t0_, t1_ = 0.0, 1.0
+    t0, t1 = torch.tensor([t0_, t1_], device=device)
+    for _ in range(REPS):
+        W = torch.tensor(npr.randn(), device=device).repeat(LARGE_BATCH_SIZE)
+        bm = torchsde.BrownianInterval(t0=t0, t1=t1, W=W, levy_area_approximation=levy_area_approximation)
+
+        for _ in range(REPS):
+            t_ = npr.uniform(low=t0_, high=t1_)
+            samples = bm(t_)
+            samples_ = samples.cpu().detach().numpy()
+
+            mean_ = W.cpu() * (t_ - t0_) / (t1_ - t0_)
+            std_ = np.sqrt((t1_ - t_) * (t_ - t0_) / (t1_ - t0_))
+            ref_dist = norm(loc=mean_, scale=std_)
+
+            _, pval = kstest(samples_, ref_dist.cdf)
+            assert pval >= ALPHA
+
+
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("levy_area_approximation", ['none', 'space-time', 'davie', 'foster'])
+def test_normality_conditional(device, levy_area_approximation):
+    if device == gpu and not torch.cuda.is_available():
+        pytest.skip(msg="CUDA not available.")
+
+    t0_, t1_ = 0.0, 1.0
+    t0, t1 = torch.tensor([t0_, t1_], device=device)
+    for _ in range(REPS):
+        bm = torchsde.BrownianInterval(t0=t0, t1=t1, shape=(LARGE_BATCH_SIZE,),
+                                       levy_area_approximation=levy_area_approximation)
+
+        for _ in range(REPS):
+            t_ = npr.uniform(low=t0_, high=t1_)
+            ta = npr.uniform(low=t0_, high=t1_)
+            tb = npr.uniform(low=t0_, high=t1_)
+            ta, t_, tb = sorted([t_, ta, tb])
+
+            increment = bm(ta, tb).cpu().detach().numpy()
+            sample = bm(ta, t_).cpu().detach().numpy()
+
+            mean = increment * (t_ - ta) / (tb - ta)
+            std = np.sqrt((tb - t_) * (t_ - ta) / (tb - ta))
+            rescaled_sample = (sample - mean) / std
+
+            _, pval = kstest(rescaled_sample, 'norm')
+            assert pval >= ALPHA
+
+
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("levy_area_approximation", ['none', 'space-time', 'davie', 'foster'])
+@pytest.mark.parametrize("random_order", [False, True])
+def test_continuity(device, levy_area_approximation, random_order):
+    if device == gpu and not torch.cuda.is_available():
+        pytest.skip(msg="CUDA not available.")
+
+    ts = torch.linspace(0., 1., 10000, device=device)
+    bm = torchsde.BrownianInterval(t0=ts[0], t1=ts[-1], shape=(), device=device,
+                                   levy_area_approximation=levy_area_approximation)
+    vals = torch.empty_like(ts)
+    i_ = torch.arange(len(ts), device=device)
+    if random_order:
+        i_ = i_[torch.randperm(len(ts), device=device)]
+    for i in i_:
+        t = ts[i]
+        vals[i] = bm(t)
+    last_val = vals[0]
+    for val in vals[1:]:
+        assert (val - last_val).abs().max() < 5e-2
+        last_val = val
+
+
+@pytest.mark.skip("BrownianInterval.to not supported")
```

Comment:
Why is this test skipped by default?

Reply:
Because of the issues with `BrownianInterval.to` as below.

Reply:
If we were to remove the `to` function, could we also remove the corresponding tests?

Reply:
Depends - do we want to leave the possibility of supporting this open in the future (in which case let's leave it in but skipped), or do we want to remove it entirely? No strong feelings either way.

Reply:
If we leave it here, could we at least put a todo?

Reply:
Sure.